src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN apt-get update -y && \
 RUN git clone --depth 1 --single-branch --branch v2.4.4 --recursive https://github.com/microsoft/msquic
 RUN cd msquic/ && \
     mkdir build && \
-    cmake -B build -DCMAKE_BUILD_TYPE=Debug -DQUIC_ENABLE_LOGGING=false -DQUIC_USE_SYSTEM_LIBCRYPTO=true -DQUIC_BUILD_TOOLS=off -DQUIC_BUILD_TEST=off -DQUIC_BUILD_PERF=off -DQUIC_TLS=openssl3 -DQUIC_ENABLE_SANITIZERS=on && \
+    cmake -B build -DCMAKE_BUILD_TYPE=Debug -DQUIC_ENABLE_LOGGING=false -DQUIC_USE_SYSTEM_LIBCRYPTO=true -DQUIC_BUILD_TOOLS=off -DQUIC_BUILD_TEST=off -DQUIC_BUILD_PERF=off -DQUIC_TLS=openssl3 && \
+      # -DQUIC_ENABLE_SANITIZERS=on && \
```

Comment:
@rzikm if this is temporary, shouldn't we open a tracking issue with a reference to https://github.com/actions/runner-images/issues/9524?

Reply:
But thanks for the reference to the runner images repo, that sounds like a useful info for future investigation